src/ROUTES.ts

```diff
@@ -739,20 +740,19 @@ export default ROUTES;
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
 type ExtractRouteName<TRoute> = TRoute extends {getRoute: (...args: any[]) => infer TRouteName} ? TRouteName : TRoute;
 
-type AllRoutes = {
+/**
+ * Represents all routes in the app as a union of literal strings.
+ * */
+type Route = {
     [K in keyof typeof ROUTES]: ExtractRouteName<(typeof ROUTES)[K]>;
 }[keyof typeof ROUTES];
 
-type RouteIsPlainString = IsEqual<AllRoutes, string>;
+type RoutesValidationError =
+    'Error: RoutesValidationError implies that one or more routes defined within `ROUTES` have not correctly used `as const` in their `getRoute` function return value.';
```

Comment:
```suggestion
type RoutesValidationError =
    'Error: One or more routes defined within `ROUTES` have not correctly used `as const` in their `getRoute` function return value.';
```

To be more aligned with the pattern [here](https://github.com/Expensify/App/blob/main/src/ONYXKEYS.ts#L660).